gateway/api/views/jobs.py

```diff
@@ -116,29 +159,26 @@ def result(self, request, pk=None):  # pylint: disable=invalid-name,unused-argum
         tracer = trace.get_tracer("gateway.tracer")
         ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
         with tracer.start_as_current_span("gateway.job.result", context=ctx):
-            saved = False
-            attempts_left = 10
-            while not saved:
-                if attempts_left <= 0:
-                    return Response(
-                        {"error": "All attempts to save results failed."}, status=500
-                    )
+            author = self.request.user
+            job = self.jobs_repository.get_job_by_id(pk)
+            if job is None:
+                logger.info("Job [%s] nor found", pk)
```

Comment:
Similar to my previous comments, how we are logging it in the repository there is no need to log it the error here too.